code/game/objects/items/mail.dm

```diff
@@ -107,26 +107,47 @@
 			var/tag = uppertext(GLOB.TAGGERLOCATIONS[destination_tag.currTag])
 			to_chat(user, span_notice("*[tag]*"))
 			sort_tag = destination_tag.currTag
-			playsound(loc, 'sound/machines/twobeep_high.ogg', 100, TRUE)
+			playsound(loc, 'sound/machines/twobeep_high.ogg', vol = 100, vary = TRUE)
+
+/obj/item/mail/multitool_act(mob/living/user, obj/item/tool)
+	. = FALSE // FALSE - means this proc was not stopped by return statement.
```

Comment:
Don't do this. `return FALSE` at the end of the proc instead.

Reply:
Also the comment is not necessary.

Reply:
Okay